ABOUT.md

```diff
@@ -1,45 +1,90 @@
+
+### Replenishment
+Your citizens need to be fed well. Keep a Farmer or an animal herder to supply the Baker and the Cook so no one goes hungry. Make sure to hire Couriers to take food (and other materials) from place to place.
+
+### Have Fun, Play with Friends
+MineColonies has a very powerful multiplayer system. The permission system allows for collaborative team play in multiplayer where you can assign different roles in a colony such as Officer, Friend, Neutral, and Hostile! You can also have competing colonies at the same time, all with a strong protection system for server management and anti-griefing. Fun for all!
+___
+MineColonies is a free and open source mod developed by Let's Dev Together (LDT), a non-profit community. The source code is available on [GitHub](https://github.com/ldtteam/minecolonies). Our developers are a hard-working, well-integrated coding team, continuously adding more content to make the MineColonies experience even greater. However, we are always looking for more people to contribute to the mod, whether as a coder, builder, artist, voice actor, wiki editor, tester, or simply supporting us on [Patreon](https://https://www.patreon.com/minecolonies)!
 
-### Have fun, Play with Friends
-Minecolonies has a very powerful multi-player system and an awesome tool for learning about Town planning, building and especially management. The permission system allows for collaborative team play in multi-player where you can assign different roles in a Town such as; Officer, Friend Neutral or even Hostile! Also have individual competing colonies at the same time, all with a strong protection system for server management and Anti-Griefing. Fun for all! 
+Found a bug? Report it as an [issue](https://https://github.com/ldtteam/minecolonies/issues/new/choose) to help us give you the best gaming experience. If you have any questions or just want to join the conversation, check us out on [Discord](https://discord.minecolonies.com)!
 ___
-##### Minecolonies is a free and open source mod developed by LDT (Let's Dev Together), a non-profit community. The source code is on [GitHub.](https://github.com/ldtteam/minecolonies) Our developers are a hardworking well integrated coding team, continuously working to add more content to our mod and make the Minecolonies experience even greater! Contribute to our Project. Found any bug? Report an [issue](https://github.com/ldtteam/minecolonies/issues/new) so as to help us ensure you have the best gaming experience. For more information and enquiries, contact us on [Discord.](https://discord.gg/YEas2Yv)
-___ 
 
 ## Our Team
 
 ### Project Leads
-* Raycoms – Programming and Management
-* Kostronor – Programming and Management
-* Asherslab – Programming and Management
-* OrionOnline - Programming and Management, Build System Engineer 
-* Pmardle – Project Management and Ideas (inactive)
+* Raycoms – Lead Lead Dev, Programming, and Active Maintainer
+* AshersLab – Programming and Active Maintainer
```

Comment:
move Asher please after Orion and Someaddons

Reply:
Sure (I only put them in that order to make it alphabetical)

Reply:
I like to have it by project arrival time.